e2e/__tests__/failures.test.js

```diff
@@ -24,21 +24,21 @@ function cleanStderr(stderr) {
 
 test('not throwing Error objects', () => {
   let stderr;
-  stderr = runJest(dir, ['throw_number.test.js']).stderr;
+  stderr = runJest(dir, ['throwNumber.test.js']).stderr;
   expect(cleanStderr(stderr)).toMatchSnapshot();
-  stderr = runJest(dir, ['throw_string.test.js']).stderr;
+  stderr = runJest(dir, ['throwString.test.js']).stderr;
   expect(cleanStderr(stderr)).toMatchSnapshot();
-  stderr = runJest(dir, ['throw_object.test.js']).stderr;
+  stderr = runJest(dir, ['throwObject.test.js']).stderr;
   expect(cleanStderr(stderr)).toMatchSnapshot();
-  stderr = runJest(dir, ['assertion_count.test.js']).stderr;
+  stderr = runJest(dir, ['assertionCount.test.js']).stderr;
   expect(cleanStderr(stderr)).toMatchSnapshot();
-  stderr = runJest(dir, ['during_tests.test.js']).stderr;
+  stderr = runJest(dir, ['duringTests.test.js']).stderr;
   expect(cleanStderr(stderr)).toMatchSnapshot();
 });
 
 test('works with node assert', () => {
   const nodeMajorVersion = Number(process.versions.node.split('.')[0]);
-  const {stderr} = runJest(dir, ['node_assertion_error.test.js']);
+  const {stderr} = runJest(dir, ['assertionError.test.js']);
```

Comment:
not sure why not `nodeAssertionError`?

Reply:
It was causing errors when I was testing it.

I wasn't really sure why but the following names passed the testing: `node_assertionError`, `node_assertion_error`, `node-assertion-error`, `AssertionNodeError`, `assertionError`. And it was failing tests when I called it: `nodeAssertionError`, `aNodeAssertionError`.

Here's the error I get with the other ones:
```
e2e/__tests__/failures.test.js (30.145s)
  ● works with node assert

    expect(string).toContain(value)

    Expected string:
      "FAIL __tests__/aNodeAssertionError.test.js
//...

      ● assert.doesNotThrow

        assert.doesNotThrow(function)

        Expected the function not to throw an error.
        Instead, it threw:
          [Error: err!]

        Message:
          Got unwanted exception.
        Actual message: \"err!\"
            at Object.doesNotThrow (C:\\Users\\Gonryun\\Desktop\\Programming\\OpenSource\\jest\\e2e\\failures\\__tests__\\aNode
"
//...

    To contain value:
      "
          69 |
          70 | test('assert.doesNotThrow', () => {
        > 71 |   assert.doesNotThrow(() => {
             |          ^
          72 |     throw Error('err!');
          73 |   });
          74 | });

          at __tests__/aNodeAssertionError.test.js:71:10
    "
```

Reply:
wierd, but I guess we can live with that (test is named correctly so why bother)